### PR TITLE
chore(prlint): should allow revert commits

### PR DESCRIPTION
**Pull request by corymhall**
----

### All Submissions:

* [ ] Have you followed the guidelines in our [Contributing guide?](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md)

### Adding new Unconventional Dependencies:

* [ ] This PR adds new unconventional dependencies following the process described [here](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md/#adding-new-unconventional-dependencies)

### New Features

* [ ] Have you added the new feature to an [integration test](https://github.com/aws/aws-cdk/blob/main/INTEGRATION_TESTS.md)?
	* [ ] Did you use `yarn integ` to deploy the infrastructure and generate the snapshot (i.e. `yarn integ` without `--dry-run`)?

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: tools/@aws-cdk/prlint/lint.ts [hunk 1]
```diff
@@ -438,7 +438,7 @@ function hasLabel(pr: GitHubPr, labelName: string): boolean {
  */
  function validateTitlePrefix(pr: GitHubPr): TestResult {
   const result = new TestResult();
-  const titleRe = /^(feat|fix|build|chore|ci|docs|style|refactor|perf|test)(\([\w_-]+\))?: /;
+  const titleRe = /^(feat|fix|build|chore|ci|docs|style|refactor|perf|test|revert)(\([\w_-]+\))?: /;
```
Comment (TheRealAmazonKendra) [inline tools/@aws-cdk/prlint/lint.ts:441]: This check is case sensitive, isn't it? Revert is capitalized in the title.

**Review by TheRealAmazonKendra — DISMISSED** [review]

File: tools/@aws-cdk/prlint/lint.ts (hunk 1 above)
Comment (TheRealAmazonKendra) [reply tools/@aws-cdk/prlint/lint.ts:441]: I see that you actually updated the title in the one that you posted earlier but wouldn't we want to not have to make people do that each time for this to pass?

Comment (corymhall) [reply tools/@aws-cdk/prlint/lint.ts]: yeah I went ahead and updated it.

**Review by Naumel — APPROVED** [review]
🔡

**Comment by Naumel** [comment]
@Mergifyio update

**Pull request merged**